### PR TITLE
run-make-check: enable --with-seastar option

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

File: run-make-check.sh [hunk 1]
```diff
@@ -61,7 +61,8 @@ function run() {
     fi
 
     if test -f ./install-deps.sh ; then
-	$DRY_RUN source ./install-deps.sh || return 1
+        export WITH_SEASTAR
```
Comment (tchaikov) [inline run-make-check.sh]: hey @wjwithagen please note, seastar does not support FreeBSD yet. if you want make FreeBSD an exception in run-make-check.sh, please let me know so i can do so here.

Comment (wjwithagen) [reply run-make-check.sh]: @tchaikov 
But looking at things, it would need some what more work... Like installing the packages in `run()`

Comment (tchaikov) [reply run-make-check.sh]: could you be more specific on what package we need to install ?

Comment (wjwithagen) [reply run-make-check.sh]: I'm not sure of the environment that this is executed in.
To run the tests atleast the packages required for tests need to be available.
And I've only started to look into that, due to the MRG/Dashboard requirements.

My comment was in genral view of the typical install case for all kinds of platforms dependant on the type of installer tools availabel. And non matches FreeBSD, but then perhaps you are right that ./install-deps.sh has done all the work already.

like I said, I'm currently not using/supporting it.

Comment (tchaikov) [reply run-make-check.sh]: @wjwithagen thanks. i just wanted to make sure my changes of run-make-check.sh does not break your CI.

Comment (wjwithagen) [reply run-make-check.sh]: @tchaikov 
Nope, it will not, so you are fine as far as FreeBSD CI

Comment (wjwithagen) [reply run-make-check.sh]: @tchaikov 
My Jenkins run also does not show any trouble, so I'll sign off.
But that is more that it doesn't cause trouble on FreeBSD.

**Comment by tchaikov** [comment]
https://shaman.ceph.com/builds/ceph/wip-22809

**Comment by tchaikov** [comment]
changelog

- add `#include "common/errno.h" in `rgw_admin_user.cc`

**Pull request merged**